app/proguard-rules.pro

```diff
@@ -73,3 +73,11 @@
 -dontwarn org.conscrypt.**
 # A resource is loaded with a relative path so the package of this class must be preserved.
 -keepnames class okhttp3.internal.publicsuffix.PublicSuffixDatabase
+
+####################################################################################################
+# AndroidX
+####################################################################################################
+
+# A reference class used by androidx.concurrent.futures.AbstractFuture$UnsafeAtomicHelper
+# Currently consumed (in depenendecies) by mozilla.components.service.freboard.scheduler
+-dontwarn sun.misc.Unsafe
```

Comment:
Nit: might be good to note that `sun.misc.Unsafe` doesn't exist in Android.

Reply:
isn't it though - https://android.googlesource.com/platform/libcore/+/9edf43dfcc35c761d97eb9156ac4254152ddbc55/libdvm/src/main/java/sun/misc/Unsafe.java?

Reply:
Huh.  StacOverflow lied to me.  Nevermind.